core/src/main/scala/org/apache/spark/scheduler/TaskDescription.scala

```diff
@@ -66,7 +100,8 @@ private[spark] object TaskDescription {
     }
   }
 
-  def encode(taskDescription: TaskDescription): ByteBuffer = {
+  @throws[TaskNotSerializableException]
+  def encode(taskDescription: TaskDescription, serializedTask: ByteBuffer): ByteBuffer = {
```

Comment:
Oh, I'm already confused, should I remove  the commit https://github.com/apache/spark/pull/15505/commits/5e03e2cf9786af6d3ea555f6f61b6a60b23f1c2c?